gravitee-apim-console-webui/src/management/api/component/health-check-v4-form/api-health-check-v4-form.component.html

```diff
@@ -0,0 +1,37 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<mat-card class="healthCheckV4">
+  <div class="apiHealthCheck__header">
+    <h2>Health-check</h2>
+    Monitor the availability and health of your endpoints and/or your API Gateways
+  </div>
+
+  <ng-container *ngIf="healthCheckForm" [formGroup]="healthCheckForm">
+    <gio-form-slide-toggle class="apiHealthCheck__enableField">
+      <gio-form-label>Enabled</gio-form-label>
+      This service is requiring an API deployment. Do not forget to deploy API to start health-check service.
+      <mat-slide-toggle
+        gioFormSlideToggle
+        formControlName="enabled"
+        aria-label="Enable health check service"
+        name="enabled"
+      ></mat-slide-toggle>
+    </gio-form-slide-toggle>
+    <gio-form-json-schema [jsonSchema]="schema" [formControlName]="'http-health-check-configuration'"></gio-form-json-schema>
```

Comment:
Here we have a static value so we can remove the [ ] I think
```suggestion
    <gio-form-json-schema [jsonSchema]="schema" formControlName="configuration"></gio-form-json-schema>
```